tests/test_schemas.py

```diff
@@ -87,7 +116,12 @@ def test_2_03_fail_files(activity_schema, filepath):
     filename_no_extension = filename.split('.')[0]
     failure_reason = filename_no_extension.split("_")[0]
 
-    error_log = iati.validator.validate_is_iati_xml(dataset, activity_schema)
+    if 'iati-activities' in filepath:
```

Comment:
As above.

Reply:
Resolved by merging `2.03dev` branch in f0aca6e.